tracing-futures/Cargo.toml

```
@@ -18,16 +18,15 @@ keywords = ["logging", "profiling", "tracing", "futures", "async"]
 license = "MIT"
 
 [features]
-default = ["futures-01", "tokio"]
-futures-01 = ["futures"]
-futures-03 = ["std-future", "futures-task"]
+default = ["std-future", "tokio"]
```

This should probably enable the _new_ tokio support instead:
```suggestion
default = ["tokio-alpha"]
```

---

Actually, strike that...the last tokio release version depends on a `futures` alpha. Why don't we just remove the tokio integration from default features until tokio 0.2 is out of alpha?